sdk/eventhub/event-hubs/src/batchingReceiver.ts

```diff
@@ -352,9 +352,9 @@ export class BatchingReceiver extends EventHubReceiver {
 
     const jitterInSeconds = randomNumberFromInterval(1, 4);
     const times =
-      retryOptions && retryOptions.retryCount && retryOptions.retryCount > 0
-        ? retryOptions.retryCount
-        : Constants.defaultRetryAttempts;
+      retryOptions && retryOptions.maxRetries && retryOptions.maxRetries > 0
+        ? retryOptions.maxRetries
+        : Constants.defaultMaxRetries;
```

Comment:
Update `times` -> `maxRetries` in retry config before passing it to `retry` method.

I wonder why it didn't give you any compile time error.

Reply:
Yes, there were some compilation issues, now fixed.